tardis/tests/integration_tests/test_integration.py

```diff
@@ -28,6 +30,7 @@ def setup(self, request, reference, data_path, atomic_data_fname):
         self.name = data_path['setup_name']
 
         self.config_file = os.path.join(data_path['config_dirpath'], "config.yml")
+        self.config_dict = yaml.load(self.config_file)
```

Comment:
`run_tardis` was proposed way back earlier but I exactly cant remember why it was turned down. For the first comment, it was to simply check the name of atom data but we dont need it yet so I am removing it.